x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/AbstractFunctionTestCase.java

```diff
@@ -1148,6 +1148,7 @@ private static void renderKibanaFunctionDefinition(
             }
             builder.endArray();
         }
+        builder.field("preview", info.preview());
```

Comment:
Nit: wondering if we could/should add `preview` only then when it's `true`. Though "explicit is better than implicit".

Reply:
I'll let Kibana folks decide here, cc @drewdaemon

Reply:
No preference here.